tests/test_login.py

```diff
@@ -12,8 +12,8 @@ class TestLogin(object):
 
     _fxa_logged_in_indicator_locator = (By.ID, 'loggedin')
 
-    def test_user_can_sign_in(self, mozwebqa, dev_account):
-        fxa = WebDriverFxA(mozwebqa)
+    def test_user_can_sign_in(self, base_url, selenium, dev_account, click_login):
+        fxa = WebDriverFxA(base_url, selenium)
         fxa.sign_in(dev_account.email, dev_account.password)
-        WebDriverWait(mozwebqa.selenium, mozwebqa.timeout).until(
+        WebDriverWait(selenium, 20).until(
```

Comment:
When I was testing locally I saw a failure once because the login was taking so long to complete that I got a timeout here with `10`. I don't think we need to change all timeouts to `20`, just this one, so I wouldn't want to add a fixture with a value of `20`.

I'm not quite as concerned with hardcoded timeouts in this project, tbh, as there are only a handful of tests and they really only exist to prove that fxapom works. I think it's fine to special case tests when needed because of that.